src/Microsoft.AspNet.Mvc.TagHelpers/ScriptTagHelper.cs

```diff
@@ -272,8 +293,15 @@ private void EnsureGlobbingUrlBuilder()
 
             foreach (var attribute in attributes)
             {
-                builder.Append(
-                    string.Format(CultureInfo.InvariantCulture, " {0}=\"{1}\"", attribute.Key, attribute.Value));
+                if (attribute.Key.Equals(SrcAttributeName, StringComparison.OrdinalIgnoreCase))
+                {
+                    AppendSrc(builder, attribute.Key, attribute.Value, escapeQuotes: false);
```

Comment:
Gonna wait to look at this file, apply the comments in `LinkTagHelper` to `ScriptTagHelper` as well and i'll look over them in the next iteration.